modules/home-manager/thunderbird.nix

```
@@ -31,7 +28,15 @@ in
     };
 
   config = lib.mkIf enable {
-    programs.thunderbird = {
+
+    assertions = [
+      {
+        assertion = correctVersion;
+        message = "catppuccin/nix thunderbird module requires at least version 25.05 of home-manager";
+      }
+    ];
+
+    programs.thunderbird = lib.mkIf correctVersion {
```

We shouldn't `mkIf` when we assert.
```suggestion
    programs.thunderbird = {
```

---

This would fail with 
```nix
error: The option `nodes.machine.home-manager.users.pepperjack.programs.thunderbird.profiles.catppuccin-mocha-mauve.extensions' does not exist. Definition values:
       - In `/nix/store/7m09k6fkcj8al93ry57x5fkq25r643h7-source/modules/home-manager/thunderbird.nix':
           [
             <derivation catppuccin-mocha-mauve.thunderbird.theme>
           ]